src/environments/environment.ts

```diff
@@ -52,7 +52,7 @@ const ADMIN_API = `${adminIP}/adminapi-v1.0/`;
 
 const IOT_API = 'http://localhost:8085/ezdx-hub-connect-srv';
 
-const FHIR_API = `${FHIRIP}/fhirapi-v1.0/`;
+const FHIR_API = `${FHIRIP}/fhirapi-v1.2/`;
```

Comment:
Please keep version as 1.0

Reply:
@devikasuresh20 shall we use APIs also without the version numbers?
`/fhirapi` 

cc: @helenKaryamsetty

Reply:
@drtechie  Yes we can use APIs without version numbers, but please note that this would require redeploying the API WAR along with updating relevant UI components and API properties.